### PR TITLE
New package: puyovs-2022.11.28_1

### PR DESCRIPTION
**Pull request by poopingpenis**
This is a Free Software game and clone of the Puyo Puyo puzzle game series.

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

File: srcpkgs/puyovs/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'puyovs'
+pkgname=puyovs
+version=2022.11.28
+revision=1
+build_wrksrc=${pkgname}-nightly
```
Comment (eli-schwartz) [inline srcpkgs/puyovs/template:5]: Highly doubtful that building a nightly version (that upstream even tells you not to use for ordinary usage) is suitable for packaging.

File: srcpkgs/puyovs/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+build_style="cmake"
+short_desc="PuyoVS"
+maintainer="poopingpenis <poopingpeniz@outlook.com"
+license="GPL-3.0"
+homepage="https://puyovs.com/"
+distfiles="https://github.com/puyonexus/puyovs/archive/refs/tags/nightly.tar.gz https://github.com/g-truc/glm/archive/refs/tags/0.9.9.8.tar.gz>glm-v0.9.9.8.tar.gz https://github.com/zlib-ng/zlib-ng/archive/refs/tags/2.0.6.tar.gz>zlib-ng-v2.0.6.tar.gz https://github.com/randy408/libspng/archive/refs/tags/v0.7.2.tar.gz>libspng-v0.7.2.tar.gz https://github.com/libsdl-org/SDL_ttf/archive/refs/tags/release-2.20.1.tar.gz>SDL_ttf-v2.20.1.tar.gz https://github.com/libsdl-org/SDL/archive/refs/tags/release-2.24.1.tar.gz>SDL-v2.24.1.tar.gz https://github.com/libsdl-org/freetype/archive/6fc77cee03e078e97afcee0c0e06a2d3274b9a29.tar.gz>freetype-6fc77cee03e078e97afcee0c0e06a2d3274b9a29.tar.gz https://github.com/libsdl-org/harfbuzz/archive/6022fe2f68d028ee178284f297b3902ffdf65b03.tar.gz>harfbuzz-6022fe2f68d028ee178284f297b3902ffdf65b03.tar.gz"
```
Comment (eli-schwartz) [inline srcpkgs/puyovs/template:11]: None of this should be using vendored copies ever, at all, for any reason. :)

**Review by eli-schwartz — CHANGES_REQUESTED** [review]

File: srcpkgs/puyovs/template (hunk 1 above)
Comment (abenson) [reply srcpkgs/puyovs/template:5]: We won't accept the package unless it is versioned appropriately.

File: srcpkgs/puyovs/template (hunk 2 above)
Comment (abenson) [reply srcpkgs/puyovs/template:11]: No vendored libraries, must be a versioned `distfile`.

File: srcpkgs/puyovs/template (hunk 1 above)
Comment (poopingpenis) [reply srcpkgs/puyovs/template:5]: I'll wait for the next release then, since the last one had too many issues.

**Pull request closed without merging**